interview-questions.md

```diff
@@ -8,89 +8,104 @@ Then, research the question to expand on your answer. Even if you feel you have
 1. Why would you use the method `super()`?
 
   Your answer:
-
+    super() is used as a method within a class. This allows the class to access the variables and data from the other class that was super()'d in.
   Researched answer:
+    super() is a method that is used in class inheritance. After a child class is extended from its parent class, super() is evoked within its constructor() to  inherit the attributes from its parent. You can also use super() to call on methods from the parent class.
 
 
 
 2. What is the DOM? What is the virtual DOM?
 
   Your answer:
-
+    The DOM is something we can manpipulate as coders to change the layout of the webpage.
```

Comment:
For sure, but let's consider that the interviewer is also asking what a "Virtual DOM" is.